core/src/event/callback_registry.rs

```diff
@@ -85,6 +86,7 @@ impl EventResult {
             log: log.clone(),
             decoded_data: network_contract.decode_log(log.inner),
             tx_information: TxInformation {
+                chain_id: network_contract.cached_provider.chain.id(),
```

Comment:
rust project will need changing to add this also

Reply:
Has been added everywhere it was needed and it works fine, i dont think anything else is needed

Reply:
awesome